pkg/usage/usage.go

```diff
@@ -222,10 +225,42 @@ func (u *usage) TriggerSingleReporter(ctx context.Context) {
 		return
 	}
 
-	logger, _ := custom_logger.GetZapLogger(ctx)
+	logger, _ := customlogger.GetZapLogger(ctx)
 
-	err := usageClient.SingleReporter(ctx, u.reporter, usagepb.Session_SERVICE_MODEL, config.Config.Server.Edition, u.version, u.userUID, u.RetrieveUsageData())
+	err := usageclient.SingleReporter(ctx, u.reporter, usagepb.Session_SERVICE_MODEL, config.Config.Server.Edition, u.version, u.userUID, u.RetrieveUsageData())
 	if err != nil {
 		logger.Fatal(err.Error())
 	}
 }
+
+type ModelUsageHandlerParams struct {
+	UserUID        uuid.UUID
+	OwnerUID       uuid.UUID
+	ModelUID       uuid.UUID
+	ModelVersion   string
+	ModelTriggerID string
+	ModelID        string
+	RequesterUID   string
```

Comment:
Requester UID should share the type with other UIDs

```suggestion
	UserUID        uuid.UUID
	RequesterUID   uuid.UUID
	OwnerUID       uuid.UUID
	ModelUID       uuid.UUID
	ModelVersion   string
	ModelTriggerID string
	ModelID        string
```

Reply:
updated